### ISSUE TITLE
Seiso UI Version Not Visible

### ISSUE THREAD

**Issue by brownc** [ExpediaDotCom/seiso-ui#256]
Guys

How are we supposed to know which version of Seiso an environment is running.

When raising my previous issue, I couldn't tell you what version we're on - which is sub-optimal.  We appear to have no Help menu anyway, or an About option with further info in, which I think would be of benefit to folks -and especially you when diagnosing issues!

Thanks

/C.

**Comment by imccunn** [comment]
Good call. I will add this. It might be beneficial too to add versions of the API's that the UI is using. For instance, I would want to see what version seiso is at and what version seiso-ui is at as well.

Thanks Chris!
